manifests/init.pp

```diff
@@ -17,6 +19,20 @@
     purge   => $config_dir_purge,
   }
 
+  if str2bool($cron_purge) {
+    file {'/etc/cron.hourly/00awstats':
+      ensure => 'absent'
```

Comment:
Would it make sense to simply this down to a variable that gets based to the ensure parameter of a single file resource?